build/safetyOutlierExplorer.js

```diff
@@ -128,6 +128,17 @@ var safetyOutlierExplorer = function (webcharts, d3$1) {
 		this.wrap.append('div').attr('class', 'multiples');
 	}
 
+	function onPreprocess() {
+		//Define x- and y-axis ranges based on currently selected measure.
+		const config = this.config;
+		const measure = this.controls.wrap.selectAll('.control-group').filter(function (d) {
+			return d.value_col && d.value_col === config.measure_col;
+		}).select('option:checked').text();
+		const measure_data = this.raw_data.filter(d => d[config.measure_col] === measure);
+		this.config.x.domain = config.x.column !== 'DY' ? d3.set(measure_data.map(d => d[config.x.column])).values() : d3.extent(measure_data, d => +d[config.x.column]);
```

Comment:
Another change like [this one](https://github.com/RhoInc/safety-outlier-explorer/pull/32#pullrequestreview-16034578) needed :( Should be able to pull the axis type off of `config.time_col[].type` and then decide whether to use `d3.extent()` or `d3.set().values()` based on that.